twilight-lavalink/src/node.rs

```diff
@@ -488,15 +509,18 @@ impl Connection {
 
         let (to_node, from_lavalink) = mpsc::unbounded_channel();
         let (to_lavalink, from_node) = mpsc::unbounded_channel();
+        let lavalink_http = HyperClient::builder(TokioExecutor::new()).build_http();
```

Comment:
Why keep TLS features and then make this plaintext-only? Also, why is there a HTTP2 feature? HTTP2 requires negotiation in the TLS handshake *or* prior knowledge,  neither of which I see you doing

Reply:
This was a change that @vilgotf did. That is why I was so confused what you were referring to.

Before:
```
let lavalink_http = HyperClient::builder(TokioExecutor::new())
            .http2_only(cfg!(feature = "http2"))
            .build_http();
```
https://github.com/twilight-rs/twilight/pull/2322#discussion_r1554630168

Please tell me what we want to do here. I would like to close this PR soon.

Reply:
This is enabled in the feature flags. The legacy client isn't clear but it is enabled here: https://docs.rs/hyper-util/latest/hyper_util/client/legacy/struct.Builder.html

Reply:
So this is still open but hinges on how to proceed.

Reply:
As I mention earlier, please add a TLS parameter to the Lavalink node that determines whether TLS (and therefore HTTP2) should be used.